libraries/jdk.lua

```diff
@@ -0,0 +1,10 @@
+-- mod-version:3
```

Comment:
No need for modversion.
Should be removed from the `nodejs` one too, but it's not too important.